workshops/workshop2023.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Workshop - 2023
+permalink: /workshops/workshop2023
+---
+
+<h1 align="center">Workshop on Federated Learning for <br/> Multiple Sclerosis Lesion Segmentation</h1>
+
+<p align="center" style="font-size:150%"><strong>To be held virtually on January 20th 2023</strong></p>
+
+## About The Workshop
+
+The Artificial Intelligence and Machine-Learning Subgroup (AIMS) of the North American Imaging in MS Cooperative (NAIMS) will be hosting a remote workshop focused on federated learning. Federated learning is a technique allowing for training of AI models across multiple servers/centers without directly sharing the underlying data. Rather than sharing the data, the model is shared between institutions instead, thereby circumventing the intrinsic challenges associated with data privacy and governance.
+
+The purpose of the workshop is to gather a critical mass of people to brainstorm on suitable solutions for MS lesion segmentation in the context of a multi-site study. There will be a series of talks given by experts, with hands-on experience in federated learning for medical image segmentation, followed by a discussion session. 
+
+Please mark it on your calendars! All levels of experience and backgrounds are welcome.
+
+## Agenda
+
+This will be a single day workshop. We hope that times will accomodate people on the west and east coast of North America.
+
+**ALL TIMES EDT**:
+
+| Time | Topic |
+|---|---|
+| 10:00 | Brief intro (what is FL, purpose of the workshop) |
+| 10:15 | Introduce Miro whiteboard: If ppl during the workshop have ideas, please enter them in the whiteboard (eg: ‘question’ category) |
```

Comment:
```suggestion
| 10:15 | Introduce Miro whiteboard: It will allow people having ideas/suggestions during the workshop to enter them in the whiteboard |
```